### DISCUSSION TITLE
16-digit password requirement

### DISCUSSION THREAD

**Discussion by devcomputer** [firefly-iii/firefly-iii#6498] [General]
It is 2022 and I do not think any service I know requires a password length of 16 characters. It is mad.
What people in the real world use is PINs, 2FA and shorter passwords.

The request has been dodged many times and users are forced to screw with the system.
https://github.com/firefly-iii/firefly-iii/issues/3895

Please be reasonable and allow shorter passwords.

![Selection_138](https://user-images.githubusercontent.com/35902641/193681493-4b8f45cd-6d12-4c9e-922f-fa8979dfaa95.jpg)

Imagine typing a 16 character password on mobile. People would rather do anything else.

**Comment by JC5** [comment]
Thank you for your feedback. 

The password length requirement will not be changed.
